indico/modules/events/timetable/legacy.py

```diff
@@ -276,10 +278,11 @@ def _get_location_data(self, obj):
         return data
 
     def _get_person_data(self, person_link):
+        person = person_link.person
         return {'firstName': person_link.first_name,
                 'familyName': person_link.last_name,
                 'affiliation': person_link.affiliation,
-                'email': person_link.person.email,
```

Comment:
should be added back when the user can manage the event (i'd set `self.can_manage_event` in `__init__` to avoid calling it over and over